apps/web-giddh/src/app/invoice/proforma/proforma-list.component.html

```diff
@@ -3,79 +3,81 @@
+                            <button *ngIf="!isCompany" class="btn btn-sky-blue m-0"
+                                (click)="goToInvoice('proformas')">{{localeData?.new_proforma}}</button>
                         </div>
+        
+                        <div *ngIf="voucherType === 'estimates'" class="btn-wrapper mr-r15">
+                            <button *ngIf="!isCompany" class="btn btn-sky-blue m-0"
+                                (click)="goToInvoice('estimates')">{{localeData?.new_estimate}}</button>
+                        </div>
+        
                     </div>
-
-
+                    <form #invoiceForm="ngForm" novalidate class="form-inline input-grp-date-range-picker position-relative">
+                            <div class="d-flex">
+                                <div class="form-group mr-r1 d-flex">
+                                    <div class=" input-group custom-datepicker" (click)="showGiddhDatepicker($event)">
+                                        <i class="icon-calender calendar-addon"></i>
+                                        <input type="text" name="selectedDateRange"
+                                            [value]="(selectedDateRangeUi) ? selectedDateRangeUi : ''"
+                                            class="giddh-datepicker date-range-picker" />
+                                    </div>
+                                    <div class="search-icon-block" (click)="openModal(template)">
+                                        <i class="icon-search"></i>
+                                    </div>
+                                </div>
+            
+            
+                            </div>
+                    </form>
                 </div>
-            </form>
-
             <!-- need to add functionality -->
             <div class="clear-filter" *ngIf="showResetAdvanceSearchIcon" (click)="resetAdvanceSearch()">
                 <button class="btn btn-filter mr-0">
                     <span class="icon-cross"></span> {{commonLocaleData?.app_clear_filter}}
                 </button>
             </div>
 
-            <div class="d-flex btn-wrapper">
-                <!-- select app proforma -->
-                <div class="select-all-proformas-on-mobile-view mt-1 mr-r15" *ngIf="selectedItems.length > 0">
-                    <div class="checkbox-img mr-1">
-                        <img src="assets/images/unchecked.png" *ngIf="!allItemsSelected" (click)="toggleAllItems(true)">
-                        <img src="assets/images/checked.png" *ngIf="allItemsSelected" (click)="toggleAllItems(false)">
-                    </div>
-                    {{commonLocaleData?.app_select_all}}
+            <div class="advance-icon cursor-pointer align-items-center btn btn-primary" (click)="toggleAdvanceSearchPopup()"
+            [tooltip]="commonLocaleData?.app_advance_search">
+                    <span class="pd-r05">Filter</span>
```

Comment:
Do add multi-lingual labels and avoid hard coded values

Reply:
done